.github/workflows/ci.yml

```diff
@@ -1,5 +1,9 @@
 name: Build
-on: [push]
+on:
```

Comment:
I am not familiar with that. Does that mean the build will run every push on the master branch and also every pull request whatever branch?

Reply:
Yes,

pull requests from other peoples accounts are not considered a a "push" so CI will never run, this makes it so if i pull request from say `blacha/topo-processor` the CI system will run.

It will also run on any push to the master branch, this does mean that just random pushes to random branches will not be CI'd until a PR is made.

Reply:
Nice!